palantirnet/butler
# Butler 23 a11y

I need another set of eyes on this to see if this output makes sense for our workflow.

**right now it's important to point out that the accessibility audit runs on the test command**

## Testing Instructions
* in your project's `package.json` set the Butler dependency to this particular branch (e.g. `"butler": "palantirnet/butler#BUTLER-23-a11y"`)
* remove the `node_modules` directory if one already exists
* run `npm install` in the project root 
* If you don't already have the scripts added to your project's `package.json`, add the following code to your project's `package.json`:

  ````
  "scripts": {
    "butler": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js",
    "develop": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js develop",
    "tests": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js test",
    "deploy": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js deploy"
  },
`````
* run `npm run tests` from the project directory
* Observe that the results of the sass linting and accessibility audit appear in the console 

## Todo before merging
- [x] Add documentation (Work being done here: https://github.com/palantirnet/butler/pull/42)
- [x] npm-shrinkwrap (See https://github.com/palantirnet/butler/pull/31#issuecomment-234922488)
